bluetooth/src/commonMain/kotlin/scanner/ScanningState.kt

```diff
@@ -217,8 +209,9 @@ sealed class ScanningState : State() {
 
         sealed class NoBluetooth constructor(scanner: BaseScanner) : Initialized(scanner) {
 
-            class Disabled internal constructor(private val scanner: BaseScanner) :
-                NoBluetooth(scanner), HandleAfterOldStateIsRemoved<ScanningState> {
+            class Disabled internal constructor(
+                private val scanner: BaseScanner
+            ) : NoBluetooth(scanner), HandleAfterOldStateIsRemoved<ScanningState> {
```

Comment:
In future please keep this type of reformatting only out of the PR, makes it hard to see changes.

Reply:
Yep, good point